examples/resources/mikrotik_dns_record/import.sh

```diff
@@ -1 +1,6 @@
-terraform import mikrotik_dns_record.record example.domain.com
+# The ID argument (*2) is a MikroTik's internal id.
+# It can be obtained via CLI:
+#
+# [admin@MikroTik] /ip dns static> :put [find where address="192.168.88.1/24"]
+# *2
+terraform import mikrotik_dns_record.record "*2"
```

Comment:
@ddelnano this is a potential breaking change since before `name` attribute was the import ID.

Reply:
I'm fine with this. Using the ID is common across the other resources and the docs will reflect that.